test/tests/functions/baselinecorrection/tests

```diff
@@ -0,0 +1,77 @@
+    []
+  []
+
+  [test_errors]
+    requirement = "The BaselineCorrection class shall appropriately invoke `mooseError()` when the input configuration is not supported."
+
+    [no_input]
+      type = RunException
+      input = no_input_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: Either `data_file` or `time_values` and `acceleration_values` must be specified exclusively."
+
+      detail = "This error shall occur since no acceleration time history was provided."
+    []
+    [extra_input]
+      type = RunException
+      input = extra_input_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: Either `data_file` or `time_values` and `acceleration_values` must be specified exclusively."
+
+      detail = "This error shall occur since the parameters which control the different input types were not provided exclusively."
+    []
+    [missing_value_param]
+      type = RunException
+      input = missing_value_param_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: Either `data_file` or `time_values` and `acceleration_values` must be specified exclusively."
+
+      detail = "This error shall occur since the `time_values` and `acceleration_values` parameters were not provided together."
+    []
+    [missing_file_param]
+      type = RunException
+      input = missing_file_param_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: A column header name was specified for the for the time data. Please specify a header for the acceleration data using the 'acceleration_name' parameter."
+
+      detail = "This error shall occur since the `time_name` and `acceleration_name` parameters were not provided together."
+    []
+    [size_mismatch]
+      type = RunException
+      input = size_mismatch_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: The length of time and acceleration data must be equal."
+
+      detail = "This error shall occur since the lengths of the time data and acceleration data are not equal"
+    []
+    [no_data]
+      type = RunException
+      input = no_data_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: The length of time and acceleration data must be > 0."
+
+      detail = "This error shall occur since no data was provided at all."
+    []
+    [no_order]
+      type = RunException
+      input = no_order_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: No values were input for parameters 'accel_fit_order', 'vel_fit_order', or 'disp_fit_order'. Please specify an integer value from 0 to 9 for at least one of these parameters."
+
+      detail = "This error shall occur since no least squares polynomials would be computed and, therefore, no adjustments would be made."
```

Comment:
no least squares polynomials can be computed and, therefore, no adjustments can be made.